examples/pc_test.py

```diff
@@ -6,7 +6,14 @@
 def tone(freq, msec):
     print('freq = {:6.1f} msec = {:6.1f}'.format(freq, msec))
 
-tune = RTTTL(songs.find('Entertainer'))
+def find(name):
+    for song in songs.SONGS:
+        song_name = song.split(':')[0]
+        if song_name == name:
+            return song
+
+
+tune = RTTTL( find('Entertainer'))
```

Comment:
So I don't understand why yu're adding a new find function to every example instead of just using the songs.find function instead (which appears to be identical).

I also noticed that you added a space betweem the opening parenthesis and find, which should be removed.